server/src/logger/logger.go

```diff
@@ -26,7 +26,8 @@ func init() {
 // EnableDevelopmentLogger constructs a development logger and overwrites the default production logger
 // this is only for development
 func EnableDevelopmentLogger() {
```

Comment:
This function is called from `main.go` if a docker environment is found. I don't comprehend how that is an indicator of the server being run in dev mode, as the server is usually called within a docker environment, isn't it?

Reply:
I think this is somewhat specific to our deployment. As we use Kubernetes for deployment and only Docker for local development.
But I agree that this should be done differently by adding a flag.